website/blog/2016-12-05-easier-upgrades.md

```diff
@@ -119,4 +119,4 @@ As a conclusion, I would say, enjoy the feature and feel free [to suggest improv
 
 ### Thank you!
 
-I would like to thank the awesome companies [Zenika](http://www.zenika.com) and [M6 Web](http://www.groupem6.fr/le-groupe_en/activites/diversifications/m6-web.html) without whom none of this would have been possible!
+I would like to thank the awesome companies [Zenika](https://www.zenika.com) and [M6 Web](https://www.groupem6.fr/le-groupe_en/activites/diversifications/m6-web.html) without whom none of this would have been possible!
```

Comment:
```suggestion
I would like to thank the awesome companies [Zenika](https://www.zenika.com) and [M6 Web (archived)](https://web.archive.org/web/20161230163633/http://www.groupem6.fr:80/le-groupe_en/activites/diversifications/m6-web.html) without whom none of this would have been possible!
```

The M6 Web link is broken (404) though found an archived snapshot :)